packages/cubejs-client-react/index.d.ts

```diff
@@ -44,6 +44,31 @@ declare module '@cubejs-client/react' {
    */
   export const CubeProvider: React.FC<CubeProviderVariables>;
 
+  /**
+   * Cube.js context consumer
```

Comment:
Would be better to add a bit more descriptive comment (eg "In case when you need access to `cubejsApi` directly you can use `CubeContext` anywhere in your app")

Reply:
added